scripts/package.sh

```diff
@@ -1,30 +1,19 @@
-#!/usr/bin/env bash
-RED='\033[0;31m'
-GREEN='\033[0;32m'
-BLUE='\033[0;34m'
-RS='\033[0m' # reset
+#!/bin/bash -e
 
-echo -e "${RS}${BLUE} Cleaning and preparing..."
-find . -iname "*.tgz" -type f | grep -v 'docs/' | xargs rm
+. scripts/common.sh
 
-echo -e "${RS}${BLUE} Linting and checking...${RS}"
+echo -e "${BLUE}Cleaning and preparing...${RS}"
+rm -rf build
+
+echo -e "${BLUE}Linting and checking...${RS}"
 scripts/lint.sh
-RET=$?
 
-echo -e "${RS}${BLUE} Packaging charts... ${RS}"
+echo -e "${BLUE}Packaging charts... ${RS}"
 helm package .
-RET=$?
-
-if [[ "${RET}" == 0 ]]; then
-    helm repo index .
+helm repo index .
 
-    echo -e "Copying chart packages and index to docs ${RS}"
-    rm -f docs/*.tgz
-    rm -f docs/*.yaml
-    mv ./*.tgz docs/
-    mv ./index.yaml docs/
+echo -e "${BLUE}Copying chart packages and index to build ${RS}"
+mkdir build
+mv *.tgz index.yaml build/
 
-    echo -e "${BLUE} Now commit and push charts and docs! ${RS}"
-else
-    echo -e "${RED}Tests failed, charts not packaged! ${RS}"
-fi
\ No newline at end of file
+echo -e "${BLUE}Now commit and push charts and docs! ${RS}"
```

Comment:
Since `build/` has been added to the `.gitignore`, what to do here is no longer obvious. The build artifacts are in `build/` and hidden from a `git status`. Also, they need to be moved into `docs/` if they are to be published to the actual repository via GitHub Pages.